### PR TITLE
Extrapolate size of sample for writing.

### PR DESCRIPTION
**Pull request by altre**
Solves #126 by taking a sample of 100 rows for calculating table size.

### CODE REVIEW

File: src/writer.jl [hunk 1]
```diff
@@ -487,13 +487,15 @@ function write_parquet(io::IO, x; compression_codec = "SNAPPY")
 
     # figure out the right number of chunks
     # TODO test that it works for all supported table
-    table_size_bytes = Base.summarysize(tbl)
+    nrows = length(Tables.rows(tbl))
```
Comment (quinnj) [inline src/writer.jl:490]:
```suggestion
    nrows = Tables.rowcount(tbl)
```
This is a mostly public Tables.jl function that only works on the result of `Tables.columns`, which `tbl` is here as we've called for above.

**Review by quinnj — APPROVED** [review]
I left one suggestion to change, but otherwise, this looks good to me.

**Review by quinnj — APPROVED** [review]
LGTM; @tanmaykm ?

**Comment by tanmaykm** [comment]
LGTM :+1:

**Pull request merged**